Custom menu translation for portuguese brazil

Wisllley opened the pull request:
I sent the translation of the customization menu to be added in the rom, in the Brazilian Portuguese language, thank you

Wisllley committed:
Update bs_strings.xml

Wisllley committed:
Update bs_strings.xml

Wisllley committed:
Update bs_strings.xml

Wisllley committed:
Update bs_strings.xml

Wisllley committed:
Update bs_strings.xml

Wisllley committed:
Update cm_strings.xml

Wisllley committed:
Update bs_strings.xml

Wisllley committed:
Add files via upload

Translation cm_strings PT-BR

scotthartbti commented:
I actually just went to pull this from your github since I couldnt merge it and noticed that packages/apps/settings already has those translations... values-pt-rBR/strings.xml already exists with the translations... I am not sure what you are trying to do? Also, the naming convention should be strings.xml... a .txt file is not going to work... ?

Wisllley committed:
Add files via upload

Wisllley committed:
Add files via upload

Wisllley commented:
Yesterday I sent all the strings in the format xml, which is the file format used, if I find any errors in the strings if possible to fix them for me, whenever I translate the strings when saving the file an error appears on the last line or the first And I could never figure out why, and I could not even sort it out.
I hope this time you can include the translations in the repository

Wisllley committed:
Add files via upload

Wisllley committed:
Add files via upload

There is an error in the last line of the file, I reviewed the original file and I saw that everything is the same, or I could not see the problem, and I wonder if you can fix it for me, I tried and did not find
